libarchive/CMakeLists.txt

```diff
@@ -264,6 +264,8 @@ IF(ENABLE_INSTALL)
           RUNTIME DESTINATION bin
           LIBRARY DESTINATION lib
           ARCHIVE DESTINATION lib)
+  INSTALL(TARGETS archive archive_static
+          EXPORT libarchive_target)
```

Comment:
CMake already comes with a builtin `FindLibArchive` module that is used to implement `find_package(LibArchive)` calls.  If upstream libarchive does start providing a CMake package, it should provide the same `LibArchive::LibArchive` target name that the find module does to help projects transition.  Some internal refactoring of libarchive's CMake build system may be helpful to match target names.